tests/test_integrations/test_symphony_summit.py

```
@@ -0,0 +1,387 @@
+                dummy_symphony_summit, action.code, action.parameters_definition
+            )
+            assert task.creds.base_url.endswith("CommonWS_JsonObjCall")
+            assert task.connection_interface == ConnectionInterfaces.REST_API
+            assert task.executable == action.code
+
+    def test_dummy_creds_action_run_injects_proxy(
+        self, dummy_symphony_summit, sample_restapi_task, monkeypatch
+    ):
+        captured = {}
+
+        def mock_request(**kwargs):
```

Ruff (0.15.20)

[warning] Missing return type annotation for private function `mock_request`

(ANN202)